From f91a09f572fcd52a3c5539c3eab1349f394a44d1 Mon Sep 17 00:00:00 2001
From: Mahesh Mahadevan <r9aadq@freescale.com>
Date: Thu, 27 Oct 2011 06:52:46 -0500
Subject: [PATCH 0519/2463] ENGR00160930-3 Changes to the MX6 defconfig

Update the defconfig to enable Sabre-lite.
One binary for MX6 ARM2 & Sabre-lite

Signed-off-by: Mahesh Mahadevan <r9aadq@freescale.com>
---
 arch/arm/configs/imx6_defconfig |   14 ++++++++++++--
 1 files changed, 12 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 32453b1..234b84c 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -263,6 +263,7 @@ CONFIG_IMX_HAVE_PLATFORM_GPMI_NFC=y
 CONFIG_IMX_HAVE_PLATFORM_IMX2_WDT=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_SNVS_RTC=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_I2C=y
+CONFIG_IMX_HAVE_PLATFORM_IMX_SSI=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_ESAI=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_UART=y
 CONFIG_IMX_HAVE_PLATFORM_MXC_EHCI=y
@@ -298,6 +299,7 @@ CONFIG_FORCE_MAX_ZONEORDER=13
 CONFIG_ARCH_MX6Q=y
 CONFIG_SOC_IMX6Q=y
 CONFIG_MACH_MX6Q_ARM2=y
+CONFIG_MACH_MX6Q_SABRELITE=y
 
 #
 # MX6 Options:
@@ -307,6 +309,7 @@ CONFIG_ISP1504_MXC=y
 CONFIG_MXC_PWM=y
 # CONFIG_MXC_DEBUG_BOARD is not set
 CONFIG_ARCH_MXC_IOMUX_V3=y
+CONFIG_ARCH_MXC_AUDMUX_V2=y
 CONFIG_IRAM_ALLOC=y
 CONFIG_CLK_DEBUG=y
 CONFIG_DMA_ZONE_SIZE=184
@@ -855,7 +858,7 @@ CONFIG_PHYLIB=y
 # CONFIG_NATIONAL_PHY is not set
 # CONFIG_STE10XP is not set
 # CONFIG_LSI_ET1011C_PHY is not set
-# CONFIG_MICREL_PHY is not set
+CONFIG_MICREL_PHY=y
 # CONFIG_FIXED_PHY is not set
 # CONFIG_MDIO_BITBANG is not set
 CONFIG_NET_ETHERNET=y
@@ -1391,7 +1394,12 @@ CONFIG_FB_MODE_HELPERS=y
 # CONFIG_FB_METRONOME is not set
 # CONFIG_FB_MB862XX is not set
 # CONFIG_FB_BROADSHEET is not set
-# CONFIG_BACKLIGHT_LCD_SUPPORT is not set
+CONFIG_BACKLIGHT_LCD_SUPPORT=y
+# CONFIG_LCD_CLASS_DEVICE is not set
+CONFIG_BACKLIGHT_CLASS_DEVICE=y
+# CONFIG_BACKLIGHT_GENERIC is not set
+CONFIG_BACKLIGHT_PWM=y
+# CONFIG_BACKLIGHT_ADP8860 is not set
 
 #
 # Display device support
@@ -1468,12 +1476,14 @@ CONFIG_SND_SOC_AC97_BUS=y
 CONFIG_SND_IMX_SOC=y
 CONFIG_SND_MXC_SOC_MX2=y
 CONFIG_SND_MXC_SOC_SPDIF_DAI=y
+CONFIG_SND_SOC_IMX_SGTL5000=y
 CONFIG_SND_SOC_IMX_CS42888=y
 CONFIG_SND_SOC_IMX_SPDIF=y
 CONFIG_SND_SOC_IMX_HDMI=y
 CONFIG_SND_SOC_I2C_AND_SPI=y
 # CONFIG_SND_SOC_ALL_CODECS is not set
 CONFIG_SND_SOC_MXC_SPDIF=y
+CONFIG_SND_SOC_SGTL5000=y
 CONFIG_SND_SOC_CS42888=y
 CONFIG_SND_SOC_MXC_HDMI=y
 # CONFIG_SOUND_PRIME is not set
-- 
1.7.7.4

